iced_layershell/src/actions.rs

```diff
@@ -82,9 +104,6 @@ pub enum LayershellCustomActions {
     ForgetLastOutput,
 }
 
-/// Please do not use this struct directly
```

Comment:
this comment should not be deleted

Reply:
yeah, i must have removed it accidentally.